admission-webhook/podpreset_crd/base/crd.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
any reason why the crd is by itsself?

Reply:
It doesnt have any controller. So no rbac rule or any other config is required.